agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -2093,7 +2093,7 @@ bool backhaul_manager::handle_1905_topology_query(ieee1905_1::CmduMessageRx &cmd
                     // the timestamp of its last association.
```

Comment:
commit message (f72a6ab22aaaf49cbf4dddd9da7d1e0d24d232b7: agent: backhaul manager: update db to save association frame for each…) headline is too long.